### PR TITLE
user gets redirected to library after creating it

### PR DESCRIPTION
**Pull request by jeanzless**
What used to happen before is that when the user creates a new Library the modal closes and leaves the user in the dashboard.

- Now with this PR the user is then redirected to their library once they've created it.

### CODE REVIEW

File: ui/src/Modals/NewLibrary/Index.jsx [hunk 1]
```diff
@@ -84,10 +87,24 @@ function NewLibraryModal(props) {
       };
 
       dispatch(newLibrary(data));
-
-      close();
     }
-  }, [close, dispatch, mediaType, name, selectedFolders]);
+  }, [dispatch, mediaType, name, selectedFolders]);
+
+  //checks if the modal is open then you redirect.
```
Comment (vgarleanu) [inline ui/src/Modals/NewLibrary/Index.jsx:93]: comment not formatted properly. Remove the new line after the comment, and add a space at the start of the comment (after the two slashes)

File: ui/src/actions/library.js [hunk 2]
```diff
@@ -111,8 +111,11 @@ export const newLibrary = (data) => async (dispatch, getState) => {
         payload: res.statusText,
       });
     }
-
-    dispatch({ type: NEW_LIBRARY_OK });
+    const payload = await res.json();
```
Comment (vgarleanu) [inline ui/src/actions/library.js:114]: add a new line before this

**Comment by rodrimarchese** [comment]
Maybe adding a loader with a few seconds will prevent from showing 'no media was found' for like 1 sec before media loads when you are redirected after creating the new library. it's a detail, not neccesary at all

**Review by rodrimarchese — APPROVED** [review]
working & changes requested made, shall we add the loader? check comment above. if not, ready to merge

**Comment by vgarleanu** [comment]
> working & changes requested made, shall we add the loader? check comment above. if not, ready to merge

Lets add the loader :)

**Comment by mental32** [comment]
marking as stale for now.

**Pull request closed without merging**